WalletWasabi/WabiSabi/Models/RoundState.cs

```diff
@@ -0,0 +1,18 @@
+using NBitcoin;
+using WalletWasabi.Crypto;
+using WalletWasabi.WabiSabi.Backend.Rounds;
+using WalletWasabi.WabiSabi.Models.MultipartyTransaction;
+
+namespace WalletWasabi.WabiSabi.Models
+{
+	public record RoundState(
+		uint256 Id,
+		CredentialIssuerParameters AmountCredentialIssuerParameters,
+		CredentialIssuerParameters VsizeCredentialIssuerParameters,
+		FeeRate FeeRate,
```

Comment:
Feerate is already in `MultipartyTransactionParameters` (sorry, I still need to finish cleanups relating to that). Note that when encryption is reintroduced, that will need to be serialized separately from the state data, because that would need to remain in cleartext.